ci-operator/step-registry/ipi/install/splay/ipi-install-splay-commands.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+DURATION=$((RANDOM % 100))s
```

Comment:
I think you need random multiples of 100 to separate the installs

Reply:
\I'm not sure how google calculates the 100s period, but the read quotas that we're hitting happen in a really short window in the installer.  I'm pretty sure what we're hitting the quota on is the early read terraform does to get the state of any potentially existing resources.  I bet offsetting these so they don't all happen at once is enough.